### PR TITLE
[CFM-1220] group content added notifications

### PR DESCRIPTION
**Pull request by anvmn**
## This PR is dependent on #1059 

https://issuetracker.amplexor.com/jira/browse/CFM-1220

### CODE REVIEW

**Comment by anvmn** [comment]
@ordavidil 
CFM talks about 2 messages, while template for one them is missing.

**Comment by anvmn** [comment]
out

File: project/profiles/capacity4more/modules/c4m/user/c4m_user_notifications/includes/c4m_user_notifications_handlers.inc [hunk 1]
```diff
@@ -179,6 +179,42 @@ function c4m_user_notifications_content_recommended($flag, $entity_id, $account,
...
+ *   Content node that was added.
+ */
+function _c4m_user_notifications_new_content_published($node) {
+  global $user;
+  $message = message_create('c4m_new_content_published');
+  c4m_message_load_tokens__c4m_new_content_published($message);
+  $body_teaser = field_view_field('node', $node, 'c4m_body', 'teaser');
+  $message->arguments['@node_teaser_body'] = strip_tags($body_teaser[0]['#markup']);
+
+  $wrapper = entity_metadata_wrapper('message', $message);
```
Comment (ordavidil) [inline project/profiles/capacity4more/modules/c4m/user/c4m_user_notifications/includes/c4m_user_notifications_handlers.inc]: @anvmn 
a lot of stuff are happening in here, please add elaborated comments, thanks.

**Comment by anvmn** [comment]
## Replaced by #1189

**Pull request closed without merging**